src/Draco.Compiler/Internal/OptimizingIr/Model/Field.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Draco.Compiler.Internal.Symbols;
+
+namespace Draco.Compiler.Internal.OptimizingIr.Model;
+internal class Field(FieldSymbol symbol, Type declaringType) : IField
```

Comment:
Seal

Reply:
for some reason this type is never used, i'll investigate it later